src/OpenTelemetry.Shared/OpenTelemetry.Shared.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;net46;net461;netstandard2.0</TargetFrameworks>
+    <Description>Shared project for OpenTelemetry .NET</Description>
+    <PackageTags>$(PackageTags);Shared</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn),1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry\OpenTelemetry.csproj" />
```

Comment:
do we need to include?

Reply:
Nice question. We may not require it now. Added it to get all common files across this repo get under this project.

Reply:
will get this removed to avoid confusion.